docs/docusaurus.config.js

```diff
@@ -139,25 +179,8 @@ const config = {
   markdown: {
     mermaid: true,
   },
-  themes: [
-    "@docusaurus/theme-mermaid",
-    ["@easyops-cn/docusaurus-search-local",
-      {
-        indexBlog: false,
-        indexDocs: true,
-        indexPages: false,
-        hashed: true,
-        highlightSearchTermsOnTargetPage: true,
-        language: ["en"],
-      }],
-  ],
-  scripts: [
-    {
-      src: "https://cdn.usefathom.com/script.js",
-      site: "NBNJSTNS",
-      defer: true,
-    },
-  ],
+  themes: ["@docusaurus/theme-mermaid"],
+  onBrokenLinks: "ignore",
```

Comment:
Should throw on broken links. We should never release a docs version with broken links. Easiest to detect that on the builds instead of trying to figure it out afterwards.